### PR TITLE
nosetests to pytest

### PR DESCRIPTION
**Pull request by asanin-epfl**

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -1,16 +1,7 @@
...
-from functools import partial
-from io import StringIO
-from pprint import pformat
-
-from nose.tools import assert_raises, ok_
-
-from morphio import Morphology, set_ignored_warning
 
 
 @contextmanager
```
Comment (adrien-berchet) [inline tests/utils.py:7]: With pytest you could just use the `tmpdir` fixture to achieve the same. So in the tests you could remove all the calls to this context manager.

**Review by adrien-berchet — COMMENTED** [review]
Thank you very much for this improvement!

File: tests/utils.py (hunk 1 above)
Comment (asanin-epfl) [reply tests/utils.py]: thank you, done

**Review by adrien-berchet — APPROVED** [review]

**Comment by asanin-epfl** [comment]
Thank you!

**Pull request merged**